docs/contributors_guide/maintenance/ruby_gem_release_procedures/Release_GitHub_Deploy_RubyGems_org.rst

```diff
@@ -69,40 +74,6 @@ from an annotated tag, you must manually release the gem.
 
 To create the releases from an annotated tag:
 
-#. Clone the component repository and checkout the development
```

Comment:
@trevor-vaughan Have you verified we do not need to do this for any of our gems?  I do not think they have been normalized.

Reply:
@lnemsick-simp The steps below will work for any gem at all. I'm bypassing all of the magic and just using the normal `gem` commands.